apps/web/src/pages/templates/components/VariantsPage.tsx

```diff
@@ -53,6 +57,10 @@ export function VariantsPage() {
     return null;
   }
 
+  if (isDelayedStep) {
+    navigate(`${basePath}/${channel}/${stepUuid}`);
+  }
```

Comment:
when entering the variants list page with a step type digest / delay do redirect to the step editor page